engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorImpl.java

```diff
@@ -747,7 +671,7 @@ private void validateCascadedValues(Object value, ValidationContext<?> context,
 			);
 
 			for ( TypeVariable<?> cascadingTypeParameterOfValueType : cascadingTypeParametersOfValueType ) {
-				ValueExtractorDescriptor extractor = valueExtractors.getCascadedValueExtractor(
+				ValueExtractorDescriptor extractor = valueExtractorManager.getTypeParameterValueExtractor(
```

Comment:
Why not just `getValueExtractor()`? It's used for type parameter constraints and cascaded validation.